docs/static_landing_page/index.html

```diff
@@ -25,7 +25,7 @@
         	<span class="navbar-toggler-icon icon-bar"></span>
       		</button>
       		<div class="collapse navbar-collapse" id="navbarResponsive">
-		        <ul class="navbar-nav pt-2 mt-1 mr-xl-auto ml-xl-2 pl-xl-1 text-center text-nowrap">
+		        <ul class="navbar-nav pt-2 mt-1 mx-xl-auto mx-xl-2 px-xl-1 text-center text-nowrap">
```

Comment:
I like your screenshot of what it looks like. I'm just not sure of what all these options do. Could you explain what mx or px (and lower down py-2) mean? If you are well-versed in this, perhaps add a comment into the HTML? I think this might end up being very useful long-term as we have to maintain this. If you don't know that's also fine. I'll have to run it locally to confirm in that case, but I can do that.

Reply:
Actually `mx`, `px` and `py` etc are built-in Bootstrap classes. We don't need comments for these, they are standard classes that are provided by bootstrap. We are already using many of these classes across the whole page and anyone who needs to read into them could look at the Bootstrap documentation or the generated CSS.

Here's a short explanation:
1. `mx` stands for `margin-x` so it will add a margin in the x-axis both on the left and the right. Similarly, you can use `mr` to specify `margin-right` and `ml` for `margin-left`. By using `mx` instead of `mr` in the line 28, we are able to center the whole nav content.
2. `px` also works in a similar fashion but for padding. `py` will specify the padding on the y-axis that is - `padding top` and `padding bottom`. I added this `py` so that when the navbar collapses, there is some vertical gap between the two icons for GitHub and Discord.

Reply:
You'd be amazed how much people without front end experience struggle to understand this code 😄 [I am very much looking at myself with this comment.]

Even just a comment to the effect of "This code uses bootstrap so check out the documentation for details <link>" goes a long way.

Thanks for explaining!